releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -1,9 +1,15 @@
 # spring-cloud-azure-feature-management
+
 [Source code][source_code] | [Package (Maven)][package] | [Samples][samples] | [Product documentation][docs]
 
 # spring-cloud-azure-feature-management-web
+
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Fix bug where feature flag names were unable to have '.' when read from a configuration file, but valid when read from Azure.
```

Comment:
How about this?

* Fix a bug where feature flag names were unable to contain dots (`.`) when they are read from a configuration file.